android/app/src/main/java/org/cryptimeleon/incentive/app/ui/basket/BasketViewModel.kt

```diff
@@ -73,12 +46,4 @@ class BasketViewModel @Inject constructor(
             }
         }
     }
-
-    fun onDiscardClicked() {
-        viewModelScope.launch {
-            withContext(Dispatchers.IO) {
-                basketRepository.discardCurrentBasket(true)
-            }
-        }
-    }
```

Comment:
why so many deleted lines in the basket view model?

Reply:
Forgot to delete it earlier